deepcell/applications/multiplex_segmentation.py

```diff
@@ -40,6 +40,100 @@
 from deepcell.model_zoo import PanopticNet
 
 
+import numpy as np
+
+
+def format_output_multiplex(output_list):
+    """Takes list of model outputs and formats into a dictionary for better readability
+
+    Args:
+        output_list: list of predictions from semantic heads
+
+    Returns:
+        formatted_dict: dictionary with predictions
+    """
+
+    formatted_dict = {'whole_cell': {'inner-distance': output_list[0],
+                                     'outer-distance': output_list[1],
+                                     'fgbg': output_list[2],
+                                     'pixelwise': output_list[3]},
+                      'nuclear': {'inner-distance': output_list[4],
+                                  'outer-distance': output_list[5],
+                                  'fgbg': output_list[6],
+                                  'pixelwise': output_list[7]}
+                      }
+
+    return formatted_dict
+
+
+def deep_watershed_subcellular(model_output, compartment='whole_cell', min_distance=10,
+                               maxima_threshold=0.1, cell_threshold=0.1, exclude_border=False,
+                               small_objects_threshold=0):
+    """Postprocess model output to generate predictions for distinct cellular compartments
+
+    Args:
+        model_output (dict): Output from deep watershed model. A dict with a key corresponding to
+            each cellular compartment with a model prediction. Each key maps to a subsequent dict
+            with the following keys entries
+            - inner-distance: Prediction for the inner distance transform.
+            - outer-distance: Prediction for the outer distance transform
+            - fgbg: prediction for the foreground/background transform
+            - pixelwise: Prediction for the interior/border/background transform.
+        compartment: which cellular compartments to generate predictions for.
+            must be one of 'whole_cell', 'nuclear', 'both'
+        min_distance (int): Minimum allowable distance between two maxima to seed cells.
+        maxima_threshold (float): Threshold for maxima used to define watershed seeds.
+        cell_threshold (float): Threshold for mask used to define allowable area for watershed.
+        exclude_border (bool): Whether to include centroid detections at the border.
+        small_objects_threshold (int): Removes objects smaller than this size.
+
+    Returns:
+        numpy.array: Uniquely labeled mask for each compartment
+
+    Raises:
+        ValueError: for invalid compartment flag
+    """
+
+    valid_compartments = ['whole_cell', 'nuclear', 'both']
```

Comment:
Generally better to ask if something is in a `set` rather than in a `list` (O(1) vs O(3)).

```suggestion
    valid_compartments = {'whole_cell', 'nuclear', 'both'}
```